packages/RatPack.package/RPTests.class/instance/testTestModelValidateJsonWithToManyAttributes.st

```diff
@@ -1,4 +1,5 @@
 test - models
 testTestModelValidateJsonWithToManyAttributes
 	
- 	self assert: ([ RPTestModel createObject: '{"var1":1 , "var2":"green", "var3":1 }'. false]on: Warning do:[true]).
\ No newline at end of file
+ 	self assert: ([ RPTestModel createObjectFrom: (Dictionary newFrom: {'var1' -> 1. 'var2' -> 'green'. 'var3' -> 1}). false]
+		on: Warning do: [true]).
```

Comment:
This test as well does not validate Json anymore.
Actually I'm not sure, if the example keys destroy the wanted effect from test. This will be always true, if the incorrectObject/Json Test is true.
Because the names are false anyway.
But I'm not sure why we need this test. If there are to many keys, than at least one of these must be not defined for the model (`RPTestmodel dataTypes keys`).